securitas/templates/group.html

```diff
@@ -1,125 +1,101 @@
-            <button class="btn btn-floating btn-small right waves-effect waves-light red modal-trigger" data-target="sponsor-modal">
-              <i class="material-icons">add</i>
-            </button>
+      <div class="col-12" data-section="members">
+        <div class="d-flex justify-content-between">
+          <div class="h4">Members</div>
+          {% if current_user_is_sponsor and sponsor_form %}
+          {# TODO: not require a double enter here maybe #}
+             <form action="{{ url_for('group_add_member', groupname=group.name) }}" method="post">
+              <div class="row">
+                <div class="col">
+                  {{ sponsor_form.hidden_tag() }}
+                  {{ sponsor_form.new_member_username(class='typeahead form-control', autocomplete="off", placeholder="add user...") }}
+                  <button class="d-none" id="submit" type="submit" tabindex="9">Submit</button>
+                </div>
+              </div>
+            </form>
           {% endif %}
-        </h4>
+          </div>
         {% if members|length == 0 %}
         <p>No members yet.</p>
         {% else %}
         <form action="{{ url_for('group_remove_member', groupname=group.name) }}" method="post">
           {{ sponsor_form.hidden_tag() }}
-          <ul class="collection">
+          <ul class="list-group">
             {% set sponsor_usernames = sponsors|map(attribute='username') %}
             {% for member in members|sort(attribute='username') %}
-              <li class="collection-item avatar">
-                <!-- TODO: .red if sponsor/admin -->
-                {% if member.mail %}
-                  <img src="{{ gravatar(member.mail, 200) }}" class="circle">
-                {% else %}
-                  <i class="material-icons circle">person</i>
-                {% endif %}
-                <a href="{{ url_for('user', username=member.username) }}">
-                  <span class="title">{{ member.username }}</span>
-                </a>
-                <p>{{ member.name }}</p>
-                <p>
-                  {% if member.username in sponsor_usernames %}
-                    sponsor
+              <li class="list-group-item d-flex justify-content-between align-items-center">
+                <div class="media align-items-center">
+                  {% if member.mail %}
+                    <img src="{{ gravatar(member.mail, 50) }}">
                   {% else %}
-                    member
+                    <i class="fa fa-user fa-2x text-muted mr-3"></i>
                   {% endif %}
-                </p>
-                {% if current_user_is_sponsor %}
-                  <div class="secondary-content">
-                    <button type="submit" class="btn-flat waves-effect waves-light red-text" name="username" value="{{ member.username }}">
-                      <i class="material-icons">remove_circle_outline</i>
-                    </button>
+                  <div class="media-body">
```

Comment:
There again, I feel like it's too close to the avatar on the left and could use some margin.

Reply:
done. in: https://github.com/fedora-infra/securitas/pull/92/commits/63a03bf4e8a20b90fb08165c89114903ab14b8a1